backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Services/Interfaces/IResourceAdministrationPoint.cs

```diff
@@ -9,6 +9,12 @@ namespace Altinn.AccessManagement.UI.Core.Services.Interfaces
     /// </summary>
     public interface IResourceAdministrationPoint
     {
+        /// <summary>
+        /// Fetches the singleRights resources, processes them with filters and search string if given and then paginates the result
+        /// </summary>
+        /// <returns>the resources that match the filters and search string corresponding to the provided page</returns>
+        Task<PaginatedList<ServiceResourceFE>> GetPaginatedSearchResults(string languageCode, string[]? resourceOwnerFilters, string? searchString, int page, int numPerPage);
```

Comment:
When seeing the description for this class and this description when searching online:

> Overall, the Resource Administration Point is responsible for centralizing the management, evaluation, and distribution of access control policies within an access control system. It provides the necessary infrastructure for administrators to define and enforce policies that govern the authorization of resource access.

I'm not really sure if the methods you created, belongs in this class. Maybe we should talk to Jon Kjetil about this

Reply:
But I'm not really sure because I see it already contains a Get method 😅  so yeah we should probably talk to Jon Kjetil tomorrow and get clearified what it this file should contain

Reply:
Yeah, I just followed the establiched convention on the file (or at least that was my intention), but I don't know if that convention was right in the first place 😅

Reply:
Since you're renaming the file in your PR, I'll just leave it as-is in mine